store.go

```diff
@@ -2256,7 +2278,7 @@ func (s *store) Version() ([][2]string, error) {
 	return [][2]string{}, nil
 }
 
-func (s *store) Mount(id, mountLabel string) (string, error) {
+func (s *store) Mount(id string) (string, error) {
```

Comment:
I'd still like to let the caller override our default, so please revert this bit.